_Classes/DeusEx/Classes/HUDRadialMenu.uc

```diff
@@ -351,7 +350,7 @@ defaultproperties
      cursorPos=(Y=-1.000000)
      focusMarkerCol=(R=255,G=255,B=255)
      focusMarkerStyle=DSTY_Translucent
-     focusMarkerTex=Texture'DeusEx.WhiteDot'
+     focusMarkerTex=Texture'RSDCrap.UserInterface.WhiteDot'
```

Comment:
If we're moving the Aug Wheel stuff to RSDCrap, then PowerIconActive and PowerIconInactive also need to be moved over.

Reply:
Gonna try that